MonoGame.Framework/Graphics/RenderTarget2D.cs

```diff
@@ -26,6 +26,8 @@ private bool SuppressEventHandlerWarningsUntilEventsAreProperlyImplemented()
 	    public RenderTarget2D(GraphicsDevice graphicsDevice, int width, int height, bool mipMap, SurfaceFormat preferredFormat, DepthFormat preferredDepthFormat, int preferredMultiSampleCount, RenderTargetUsage usage, bool shared, int arraySize)
 	        : base(graphicsDevice, width, height, mipMap, preferredFormat, SurfaceType.RenderTarget, shared, arraySize)
 	    {
+            graphicsDevice.Adapter.QueryRenderTargetFormat(graphicsDevice.GraphicsProfile, preferredFormat, preferredDepthFormat, preferredMultiSampleCount,
+                out preferredFormat, out preferredDepthFormat, out preferredMultiSampleCount);
```

Comment:
QueryRenderTargetFormat() is called after the base class (Texture2D) is initialized with the preferredFormat.

Reply:
We can move this code into the private Texture2D(...) and perform the fallback when SurfaceType ==  RenderTarget. 

We also need to remove the SurfaceFormat & SurfaceType parameters from Texture2D.PlatformConstruct()
https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture2D.DirectX.cs#L37-L41
When is needed, it has to use the ._format that is already initialized from Texture2D().